user-guide/Basic_Functionality/Visio/turning_shapes_into_controls/Making_a_shape_filter_Alarm_Console_tabs_when_clicked.md

```diff
@@ -39,6 +41,8 @@ Add a shape data field of type AlarmFilter to the shape, and enter an alarm filt
   "Service Impact"<2
   ```
 
+- Example dynamic value
+
```

Comment:
Example "[var:myAlarmFilterVariable]"

Reply:
"This will, when the user clicks on the shape, attach the filter that is currently in the provided session variable."
So what it does is the same as the above listed examples, only it allows the user to dynamically set its value